assets/src/svg/individual/info.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M12 3.2c-4.8 0-8.8 3.9-8.8 8.8 0 4.8 3.9 8.8 8.8 8.8 4.8 0 8.8-3.9 8.8-8.8 0-4.8-4-8.8-8.8-8.8zm0 16c-4 0-7.2-3.3-7.2-7.2C4.8 8 8 4.8 12 4.8s7.2 3.3 7.2 7.2c0 4-3.2 7.2-7.2 7.2zM11 17h2v-6h-2v6zm0-8h2V7h-2v2z" /></svg>
+<svg version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m12 0.19202c-6.4084 0-11.749 5.2331-11.749 11.808 0 6.4407 5.2069 11.808 11.749 11.808 6.4084 0 11.749-5.2331 11.749-11.808 0-6.4407-5.3404-11.808-11.749-11.808zm0 21.469c-5.3404 0-9.6127-4.428-9.6127-9.6611 0-5.3673 4.2723-9.6611 9.6127-9.6611 5.3404 0 9.6127 4.428 9.6127 9.6611 0 5.3673-4.2723 9.6611-9.6127 9.6611zm-1.3351-2.952h2.6702v-8.0509h-2.6702zm0-10.735h2.6702v-2.6836h-2.6702z" stroke-width="1.3384"/></svg>
```

Comment:
Is the icon the same? The file looks a lot different, I'm wondering what the change was.

Reply:
I slightly reduced the padding of the image, it was too small inside buttons. 

## New
![image](https://user-images.githubusercontent.com/90911997/236560364-0d2246c6-13c4-4b46-9358-70973784eb61.png)

## Old
![image](https://user-images.githubusercontent.com/90911997/236560432-544758cf-36fd-4370-9504-365bd09c305e.png)